aspnetcore/mvc/views/partial/sample/PartialViewsSample/Pages/DiscoveryRP.cshtml

```diff
@@ -0,0 +1,118 @@
+﻿@page
+@model DiscoveryModel
+@using PartialViewsSample.ViewModels
+@{
+    ViewData["Title"] = "Partial view discovery (Razor Pages)";
+    ViewData["index"] = 0;
+}
+
+<h1>@ViewData["Title"]</h1>
+
+<table class="table table-hover">
+    <caption>Asynchronous partial view discovery examples</caption>
```

Comment:
Maybe drop the `Asynchronous`? It's an implementation detail that users don't need to know?